### PR TITLE
Count set bits of number - python

### PR DESCRIPTION
**Pull request by shahina-bano**
Added code for Count Set Bits of number for issue #44

### CODE REVIEW

File: Algorithm/Searching_Sorting/readme.md [hunk 1]
```diff
@@ -68,3 +68,4 @@ Following are the types of searches which we will be discussing in this book.
 * Shell Sort ----> [C++](/Code/C++/shell_sort.cpp)
 * Selection Sort ----> [C++](/Code/C++/selection_sort.cpp)
 
+
```
Comment (plazzy99) [inline Algorithm/Searching_Sorting/readme.md:71]: delete this redundant edit

File: Algorithm/Bit_manipulation/readme.md [hunk 2]
```diff
@@ -52,3 +52,4 @@ There are different bitwise operations used in the bit manipulation. Bit operati
 
 
 ## Questions :
+* Count set bits of a number ----> [Python](/Code/Python/setbits.py)
```
Comment (plazzy99) [inline Algorithm/Bit_manipulation/readme.md:55]: Wrong name of program

**Review by plazzy99 — CHANGES_REQUESTED** [review]

**Comment by shahina-bano** [comment]
I have resolved the conflict, please check it now.

File: Algorithm/Searching_Sorting/readme.md [hunk 3]
```diff
@@ -68,4 +68,3 @@ Following are the types of searches which we will be discussing in this book.
 * Shell Sort ----> [C++](/Code/C++/shell_sort.cpp)
 * Selection Sort ----> [C++](/Code/C++/selection_sort.cpp)
 * Tim Sort ----> [C++](/Code/C++/tim_sort.cpp)
-
```
Comment (plazzy99) [inline Algorithm/Searching_Sorting/readme.md:71]: remove this edit

**Review by plazzy99 — CHANGES_REQUESTED** [review]

**Comment by plazzy99** [comment]
Resolve conflict

**Comment by shahina-bano** [comment]
@plazzy99 actually I am facing some issue in this PR while editing so I am closing this PR and open a new PR for the same code. Hope it's not a problem.

**Comment by plazzy99** [comment]
> @plazzy99 actually I am facing some issue in this PR while editing so I am closing this PR and open a new PR for the same code. Hope it's not a problem. 

Not a problem but learn to fix this. You can it learn youtube.

**Comment by shahina-bano** [comment]
The problem was in branch so that's why, although I have resolved it .. thank you :)

**Pull request closed without merging**